tenant/storage_bucket.go

```diff
@@ -120,7 +120,6 @@ func (s *Store) GetBucketByName(ctx context.Context, tx kv.Tx, orgID influxdb.ID
 		switch n {
```

Comment:
Same for this one, the `if kv.IsNotFound(err) {` need only be `return nil, ErrBucketNotFoundByName(n)` as the fixed system buckets are gone.